Remove windows code from ews

aj062 opened the pull request:
#### 1354cb8a5c5e1ab8a6b2ca80a9ca27eb1e1822f0
<pre>
Remove windows code from ews
<a href="https://bugs.webkit.org/show_bug.cgi?id=251886">https://bugs.webkit.org/show_bug.cgi?id=251886</a>
rdar://105148896

Reviewed by Ryan Haddad and Jonathan Bedard.

As announced on webkit-dev, we are decommissioning the Apple Windows port.
We already removed the queues. We should remove the code specific to windows from ews.

* Tools/CISupport/ews-build/factories.py:
(WindowsFactory): Deleted.
(WindowsFactory.__init__): Deleted.
* Tools/CISupport/ews-build/loadConfig.py:
* Tools/CISupport/ews-build/loadConfig_unittest.py:
(TagsForBuilderTest.test_getTagsForBuilder):
* Tools/CISupport/ews-build/steps.py:
(CheckChangeRelevance):
(CompileWebKit.start):
(DeleteStaleBuildFiles): Deleted.
(DeleteStaleBuildFiles.__init__): Deleted.
* Tools/CISupport/ews-build/steps_unittest.py:
(TestCheckChangeRelevance.test_relevant_windows_wk1_patch): Deleted.

Canonical link: <a href="https://commits.webkit.org/259984@main">https://commits.webkit.org/259984@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ed37c52e9bd150989f974f3f58e4a5af65e85ca3

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [  ~~🧪 style~~](https://ews-build.webkit.org/#/builders/6/builds/106650 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/77/builds/15641 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/39437 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/8/builds/115836 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/115448 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [  ~~🧪 bindings~~](https://ews-build.webkit.org/#/builders/11/builds/110557 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/76/builds/17155 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/85/builds/6877 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/98841 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [  ~~🧪 webkitperl~~](https://ews-build.webkit.org/#/builders/19/builds/112420 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/76/builds/17155 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/39437 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/98841 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/76/builds/17155 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/39437 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/36/builds/98841 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [  ~~🛠 🧪 jsc~~](https://ews-build.webkit.org/#/builders/1/builds/96202 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/81/builds/8867 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/39437 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/82/builds/9425 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/85/builds/6877 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/20/builds/105466 "Passed tests") | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/80/builds/15040 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/39437 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/79/builds/10947 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [❌ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/3713 "Failed to push to pull request branch") | | | | 
<!--EWS-Status-Bubble-End-->

ryanhaddad reviewed (approved):
r=me assuming tests are happy.

aj062 committed:
Remove windows code from ews
https://bugs.webkit.org/show_bug.cgi?id=251886
rdar://105148896

Reviewed by Ryan Haddad and Jonathan Bedard.

As announced on webkit-dev, we are decommissioning the Apple Windows port.
We already removed the queues. We should remove the code specific to windows from ews.

* Tools/CISupport/ews-build/factories.py:
(WindowsFactory): Deleted.
(WindowsFactory.__init__): Deleted.
* Tools/CISupport/ews-build/loadConfig.py:
* Tools/CISupport/ews-build/loadConfig_unittest.py:
(TagsForBuilderTest.test_getTagsForBuilder):
* Tools/CISupport/ews-build/steps.py:
(CheckChangeRelevance):
(CompileWebKit.start):
(DeleteStaleBuildFiles): Deleted.
(DeleteStaleBuildFiles.__init__): Deleted.
* Tools/CISupport/ews-build/steps_unittest.py:
(TestCheckChangeRelevance.test_relevant_windows_wk1_patch): Deleted.

Canonical link: https://commits.webkit.org/259984@main
